config/install/user.role.az_content_admin.yml

```diff
@@ -33,7 +33,6 @@ permissions:
   - 'bypass node access'
   - 'create content translations'
   - 'create new books'
-  - 'create az_custom_menu_block block content'
```

Comment:
FWIW, the new `create xxxx block content` permissions provided by core as of Drupal 10.1 has the same machine names so I'll probably be adding this one back in my PR for #3014.